Document process to configure and update docker agent (#62)
nginxinc/docker-nginx-controller: README.md

```

For NAP-Enabled NGINX Docker Containers, information about App Security requirements can be found [here](https://docs.nginx.com/nginx-controller/admin-guides/install/try-nginx-controller-app-sec/)

### 4.7 Configuring and updating agent

> When manually configuring changes in agent.conf or upgrading NGINX Agent service running inside the container use `--restart always` during docker run as restarting the agent service will trigger container to exit.

Reviewer: ```suggestion
When manually editing the `agent.conf` file or upgrading the NGINX Agent service running inside the container, use `--restart always` with `docker run`, as restarting the NGINX Agent service will trigger the container to exit.
```
